test/api/test_issue_routes.py

```diff
@@ -6,233 +6,233 @@ def metrics():
+    response = requests.get('http://localhost:5000/api/unstable/repo-groups/10/issue-throughput')
     data = response.json()
     assert response.status_code == 200
     assert len(data) >= 1
     assert data[0]['throughput'] >= 0
 
 def test_issue_throughput_by_repo(metrics):
-    response = requests.get('http://localhost:5000/api/unstable/repo-groups/24/repos/21682/issue-throughput')
+    response = requests.get('http://localhost:5000/api/unstable/repo-groups/10/repos/25435/issue-throughput')
     data = response.json()
     assert response.status_code == 200
     assert len(data) >= 1
     assert data[0]['throughput'] >= 0
 
 def test_issue_backlog_by_group(metrics):
-    response = requests.get('http://localhost:5000/api/unstable/repo-groups/20/issue-backlog')
+    response = requests.get('http://localhost:5000/api/unstable/repo-groups/10/issue-backlog')
     data = response.json()
     assert response.status_code == 200
     assert len(data) >= 1
     assert data[0]['issue_backlog'] > 0
 
 def test_issue_backlog_by_repo(metrics):
-    response = requests.get('http://localhost:5000/api/unstable/repo-groups/20/repos/21403/issue-backlog')
+    response = requests.get('http://localhost:5000/api/unstable/repo-groups/10/repos/25435/issue-backlog')
     data = response.json()
     assert response.status_code == 200
     assert len(data) >= 1
     assert data[0]['issue_backlog'] > 0
 
 def test_issues_first_time_opened_by_group(metrics):
-    response = requests.get('http://localhost:5000/api/unstable/repo-groups/24/issues-first-time-opened')
+    response = requests.get('http://localhost:5000/api/unstable/repo-groups/10/issues-first-time-opened')
     data = response.json()
     assert response.status_code == 200
     assert len(data) >= 1
     assert data[0]["count"] > 0
 
 def test_issues_first_time_opened_by_repo(metrics):
-    response = requests.get('http://localhost:5000/api/unstable/repo-groups/24/repos/21222/issues-first-time-opened')
+    response = requests.get('http://localhost:5000/api/unstable/repo-groups/10/repos/25435/issues-first-time-opened')
     data = response.json()
     assert response.status_code == 200
     assert len(data) >= 1
     assert data[0]["count"] > 0
 
 def test_issues_first_time_closed_by_group(metrics):
-    response = requests.get('http://localhost:5000/api/unstable/repo-groups/24/issues-first-time-closed')
+    response = requests.get('http://localhost:5000/api/unstable/repo-groups/20/issues-first-time-closed')
     data = response.json()
     assert response.status_code == 200
     assert len(data) >= 1
     assert data[0]["count"] > 0
 
-def test_issues_first_time_closed_by_repo(metrics):
-    response = requests.get('http://localhost:5000/api/unstable/repo-groups/24/repos/22997/issues-first-time-closed')
-    data = response.json()
-    assert response.status_code == 200
-    assert len(data) >= 1
-    assert data[0]["count"] > 0
+# def test_issues_first_time_closed_by_repo(metrics):
```

Comment:
Why is this test commented out? If we do not need it anymore than we should delete it, if we need to reference it in the future, then we can look at commit history